Dockerfile

```diff
@@ -158,14 +162,12 @@ RUN --mount=type=cache,target=/root/.cache/microdnf:rw \
 # need to upgrade pip and install wheel before installing grpcio, before installing tensorflow on aarch64
 # use caching to speed up multi-platform builds
 ENV PIP_CACHE_DIR=/root/.cache/pip
-RUN --mount=type=cache,target=/root/.cache/pip \
-    pip install --upgrade pip && \
-    pip install wheel && \
-    pip install grpcio && \
+RUN pip install grpcio && \
     # pin to 3.10.0 to avoid error: libhdf5.so: cannot open shared object file: No such file or directory \
     # if not version is set, it will install the 3.11.0 version which, seems that does not have the h5py dependencies \
     # for arm yet.
     pip install h5py==3.10.0 && \
+    pip install tensorflow-io-gcs-filesystem==0.34.0 && \
```

Comment:
Where does this library information come from?

Reply:
as this project does not contain a explicit list of supported dependencies, all its have is TensorFlow and a few others defined in the Dockerfile, I built it and collected the version information to create the `pyptoject.toml` file so we can generate the `requirements.txt` from there.

Reply:
This has been building and working fine without this, no? If so, I don't think the generation of the requirements.txt file can justify adding it.

Reply:
on Konflux, no, it was complaining at some point, If we don't pin to 0.34 it will install 0.35 leading the build to fail.

Reply:
See my comment here: https://github.com/opendatahub-io/modelmesh-runtime-adapter/pull/65/files#r1747517727

Reply:
not sure how your comment there relates to it.